zeus/run/dataloader.py

```diff
@@ -741,46 +730,31 @@ def _is_train(self) -> bool:
         """Return whether this dataloader is for training."""
         return self.split == "train"
 
+    @property
+    def _power_log_prefix(self) -> str:
+        """Build the prefix for the power monitor log file."""
+        return f"{self.logdir}/bs{self.train_batch_size}+e{self.epoch_num}"
+
     def _power_log_path(self, rank: int) -> str:
         """Build the path for the power monitor log file at the GPU with rank."""
         return f"{self.logdir}/bs{self.train_batch_size}+e{self.epoch_num}+gpu{rank}.power.log"
 
-    def _start_monitor(self) -> None:
-        """Start the power monitor subprocess."""
-        # Sanity checks.
-        assert not ZeusDataLoader.monitors
-        # Only the master process starts the monitors.
-        assert self.rank == 0
-
-        # Start monitors. Master process starts and records all monitors.
-        for index in range(self.world_size):
-            monitor = subprocess.Popen(
-                [
-                    self.monitor_path,
-                    self._power_log_path(index),
-                    "0",
-                    "100",
-                    str(index),
-                ],
-            )
-            self._log(f"[GPU_{index}] Zeus monitor started.")
-            ZeusDataLoader.monitors.append(monitor)
+    @property
+    def _get_prof_service(self) -> ZeusProfilingService:
```

Comment:
This method name is suitable for a function, not a property. If this is to be a property, name it `_prof_service`. Otherwise, remove the property decorator and make it a function.